scripts/cjdns-iptunnel/cjdns-setup

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -e
+sleep 8
```

Comment:
I don't think this fix was necessary in my case. Can we check if these two steps are sufficient?
https://github.com/tomeshnet/prototype-cjdns-pi/issues/219#issuecomment-429618886

Really don't want a `sleep <arb value>` unless we must. And if we really must then this needs a comment to explain.

Reply:
Im doing its "just in case" because we need tun0 to be up before proceeding.

i can drop it but.... its a precaution